bin/s2nd.c

```diff
@@ -355,17 +355,17 @@ int handle_connection(int fd, struct s2n_config *config, struct conn_settings se
     }
 
     if (negotiate(conn, fd) != S2N_SUCCESS) {
+        if (settings.mutual_auth) {
+            if (!s2n_connection_client_cert_used(conn)) {
+                print_s2n_error("Error: Mutual Auth was required, but not negotiated");
+                S2N_ERROR_PRESERVE_ERRNO();
```

Comment:
`S2N_ERROR_PRESERVE_ERRNO` isn't necessary here -- you can just fall through to line 366.

Reply:
Removed